content/en/docs/demo/docker-deployment.md

```diff
@@ -39,6 +39,19 @@ cSpell:ignore: otelcollector otlphttp spanmetrics
     > `docker compose up --force-recreate --remove-orphans --detach`[^1] to
     > start the demo.
 
+4.  (Optional) Use make to enable API testing with observability-driven
+    development (ODD):
+
+    ```shell
+    make start-odd
+    ```
+
+    > **Notes:**
+    >
+    > If you do not have the make utility installed, you can also use
+    > `docker compose --profile odd up --force-recreate --remove-orphans --detach`[^1]
+    > to start the demo and the. It uses the `odd` Docker Compose Profile.
```

Comment:
I think that it would make sense to use tabbed panes here instead, one tab for **Make** and the other for **Docker**. @svrnm et al., WDYT?

If so, step 3 should be updated as well.

Reply:
Other notes:

- If you keep the note, change the label to **Note**
- Drop the copy-paste sentence fragment after the `[^1]` -- "to start the demo and the."

Reply:
Yes, tabbed panes might be a good choice here to make things look a little bit better, otherwise we get an overflow of notes

Reply:
I like this idea. I've added a commit with a draft of what it would look like. Would love some feedback.

Reply:
thanks!